### PR TITLE
Fix for issue #2204

### PR DESCRIPTION
**Pull request by TeoTwawki**
Explanation: the reason we null TextIDs.lua is because with every zone loading the same file name, you can wind up with null or invalid text ID from the previous zone being loaded instead the correct TextID from the zone a player is currently in.

### CODE REVIEW

File: scripts/zones/Norg/npcs/Muzaffar.lua [hunk 1]
```diff
@@ -1,36 +1,37 @@
...
 -- Area: Norg
--- NPC: Muzaffar
+--  NPC: Muzaffar
 -- Standard Info NPC
 -- Quests: Black Market
--- @zone 252
--- @pos 16.678, -2.044, -14.600
+-- @pos 16.678, -2.044, -14.600 252
+-----------------------------------
+package.loaded["scripts/zones/Castle_Zvahl_Baileys/TextIDs"] = nil;
```
Comment (bendangelo) [inline scripts/zones/Norg/npcs/Muzaffar.lua]: Isn't this supposed to be Norg?

Comment (TeoTwawki) [inline scripts/zones/Norg/npcs/Muzaffar.lua]: woops, failboat, amending

Comment (TeoTwawki) [inline scripts/zones/Norg/npcs/Muzaffar.lua]: I was distracted by stupid

**Comment by TeoTwawki** [comment]
@bendangelo fixed my copy paste derp

**Comment by bendangelo** [comment]
thanks

**Pull request merged**